rules/windows/powershell/powershell_script/posh_ps_create_local_user.yml

```diff
@@ -18,7 +18,9 @@ logsource:
     definition: 'Requirements: Script Block Logging must be enabled'
 detection:
     selection:
-        ScriptBlockText|contains: 'New-LocalUser'
+        ScriptBlockText|contains:
+            - 'New-LocalUser'
+            - 'nlu'
```

Comment:
These small strings can and will introduce FPs, hence why they were not added in the first place. Remember that this is targeting a ScriptBlock which contains hundreds to thousands lines of codes that start with nlu or other

Example include the word `unlucky` ;)